client/src/components/CRM/Contacts/ContactPopUp.tsx

```diff
@@ -34,6 +38,17 @@ const ContactPopUp = (props: ContactPopUpProps): ReactElement => {
     values,
   } = props;
 
+  // Assigns seatingAcc to comments input if there's a custom accommodation
+  const contact = {
```

Comment:
Is it possible to split the seating accommodation by the ' - ' so that the correct option can be selected upon load and the option itself is not in the comments edited by the user? Currently if the existing option is "Wheel Chair - ..."  and the user changes the option to Aisle the new seating accommodation will be "Aisle - Wheel Chair - ...".

Reply:
Yeah, I think it probably makes the most sense to just create a `comments` column in the schema itself at this point, rather than keep reusing `seatingAccom` for not-related circumstances.

Reply:
That sounds good to me!

Reply:
This PR grew quite a bit, but added the `comments` field

Reply:
+1 for adding the comments field. I had concatenated the seating accommodation with the comments as a short term stop gap for recording any comments that users left in the checkout process since other wise they would have been ignored